tests/bench/obex.py

```
@@ -0,0 +1,643 @@
+    print_results(agg, list(results))
+
+    print(
+        f"\nTotal time: {total_elapsed:.1f}s ({total_elapsed / len(cases):.2f}s per case)"
+    )
+
+    # Determine output file path
+    if output:
+        output_file = Path(output)
+    else:
+        # Generate default timestamped filename
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)